nf_core/pipeline-template/{{cookiecutter.pipeline_slug}}/main.nf

```diff
@@ -159,6 +159,41 @@ if(params.email) summary['E-mail Address'] = params.email
 log.info summary.collect { k,v -> "${k.padRight(15)}: $v" }.join("\n")
 log.info "========================================="
 
+
+def create_workflow_summary(summary) {
+
+    def yaml_file = workDir.resolve('workflow_summary_mqc.yaml')
+    yaml_file.text  = """
+    id: '{{ cookiecutter.pipeline_name }}-summary'
+    description: " - this information is collected when the pipeline is started."
+    section_name: 'nfcore/{{ cookiecutter.pipeline_name }} Workflow Summary'
```

Comment:
I think it should be pipeline_slug instead of pipeline_name.

cf: https://github.com/nf-core/tools/blob/dev/nf_core/pipeline-template/cookiecutter.json